src/plugins/data/public/ui/query_string_input/query_bar_top_row.test.tsx

```diff
@@ -117,7 +117,7 @@ describe('QueryBarTopRowTopRow', () => {
     jest.clearAllMocks();
   });
 
-  it('Should render the given query', () => {
+  it.skip('Should render the given query', () => {
```

Comment:
Here's the stack trace of the unhandled promise rejection:

```
TypeError: Cannot read property 'autocomplete' of undefined
    at QueryStringInputUI._defineProperty [as getSuggestions] (./src/plugins/data/public/ui/query_string_input/query_string_input.tsx:89:55)
    at QueryStringInputUI._defineProperty (./src/plugins/data/public/ui/query_string_input/query_string_input.tsx:141:39)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

Reply:
It's the same error for the other skipped tests in this file as well

Reply:
@lizozom I'm not sure if you're the right person to ping about this (blame `git blame` 😅), but I had to skip these tests as they were producing unhandled promise rejections due to an uncaught exception.

Reply:
@lukeelmers @streamich I just saw that Liza is on vacation this week, would any of you be able to help me fix these tests in this PR?

We previously discussed fixing it in a separate PR, but I think it would speed things up if you could push a commit directly to this PR so we don't block it. Any help would be much appreciated 😃